packages/esphome/desk_height_sensor.h

```diff
@@ -64,6 +64,10 @@ class DeskHeightSensor : public Component, public UARTDevice, public Sensor
     {
       return 10;
     }
+    if (b[0] && !b[1] && !b[2] && b[3] && b[4] && b[5] && b[6])
+    {
+      return 11; // letter E
```

Comment:
Personally I am not a big fan of "hiding" information in non-intuitive return value, but I wanted to keep things consistent here